pyproject.toml

```diff
@@ -28,6 +25,9 @@ pythonVersion = "3.10"
 black = "^23.1.0"
 pre-commit = "^3.0.4"
 pyright = "^1.1.292"
+pytest = "^7.2.1"
+pytest-playwright = "^0.3.0"
+pytest-base-url = "^2.0.0"
```

Comment:
Ah damn, I think I did not add pytest-django as dev dependency too. Can you change that as well?
I try to think about it next time!